application/src/main/java/org/togetherjava/tjbot/commands/utils/MessageUtils.java

```diff
@@ -56,4 +60,31 @@ public static String escapeMarkdown(String text) {
         return afterEscape.replace("\\```", "\\`\\`\\`");
     }
 
+    /**
+     * Converts a slash command text to a mentioned slash command, which you can directly click on
+     * in Discord.
+     *
+     * @param guild the {@link Guild} that contains the command
+     * @param baseCommandName the command's name
+     * @param subcommands optional subcommands, depending on the base command used
+     * @return Formatted string for the mentioned slash command
+     * @throws IllegalArgumentException when the command name doesn't match with the guild's
+     *         commands
+     */
+    public static RestAction<String> mentionSlashCommand(Guild guild, String baseCommandName,
+            String... subcommands) {
```

Comment:
y u taking multiple subcommands?

Reply:
a command can only have one subcommand no? i mean a mentioned command can only have one subcommand, then y multiple???

Reply:
A Subcommand group can have Subcommands itself. For the `help-thread` command, there is a `change` group, that has `category` and `title` bound to it. However, it also has a `close` subcommand, so here's a list of possible outcomes :
`/help-thread change category`
`/help-thread change title`
`/help-thread close`

That's why you can have a list of "subcommands"

Reply:
so `change` is actually a subcommandgroup not subcommand, what u should do is take only one input as `commandPath` which is the full path, this is also the name jda uses so it'll be a better option then u'll get `help-thread change category` as one input then u can split it and do ur thing.

Reply:
looking at ur implementation it should be an easy change, other things looks good to me

Reply:
kinda agree. its a bit hard to give this a nice and non-confusing signature without going the `fullCommand` approach. Maybe its indeed easier to just go that route instead.